packages/debug/src/browser/editor/debug-editor-model.ts

```diff
@@ -150,6 +150,26 @@ export class DebugEditorModel implements Disposable {
         return decorations;
     }
 
+    protected async toggleExceptionWidget(): Promise<void> {
+        const { currentFrame } = this.sessions;
+        if (!currentFrame ||
+            !currentFrame.source || currentFrame.source.uri.toString() !== this.uri.toString() ||
+            !currentFrame.raw.line || !currentFrame.raw.column) {
```

Comment:
This code will fail if line or column are `0` (falsey values)

```suggestion
            currentFrame.raw.line === undefined || currentFrame.raw.column === undefined) {
```